rpc/common/moc_common_common.proto

```diff
@@ -56,6 +56,7 @@ enum HealthState {
 	DEGRADED = 5;
 	// The entity went missing from the agent
 	NOTFOUND = 6;
+	NOTREADY = 7;
```

Comment:
Is there a doc to describe what each health state means for zones?

Reply:
Yes this is defined in our zones feature design doc. 

https://github.com/microsoft/wssdagent/blob/master/doc/features/zones/Zone_Management_in_moc.md

![image](https://github.com/user-attachments/assets/4c08b413-b339-4643-add0-c8f9dfdb9c1d)